app/controllers/user.js

```diff
@@ -44,6 +46,61 @@ exports.create = (req, res, next) => {
   
 };
 
+exports.signin = (req, res, next) => {
+
+  try {
```

Comment:
I would call here the `validateLoginInput` function here instead of using a middleware in the routes. I think it would be easier to follow the flow.